### DISCUSSION TITLE
Who is "For You" actually for?

### DISCUSSION THREAD

**Discussion by asmaloney** [community/community#54469] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It's clearly not for developers.

Please give us a way to _completely_ turn it off.

**Comment by ghost** [comment]
I agree; the first time I saw the board I thought "Oh no, github Facebook". Who cares who followed what and what repo is trending? For instance, I know well spring libs, and absolutely I do not need to see them anywhere because I can always find them easily. I think GitHub tries to be more inclusive for interns and people who are not programmers and can't find what they are looking for. Basically, I really wish I will be able to deactivate "For you" board totally in settings somewhere. I only need "Following" to know who and what contributes to my repos.

**Comment by ArjixWasTaken** [comment]
> It's clearly not for developers.

From my understanding, the "For you" feed brings results from the people you follow, and people that they follow or are followed by.
Logically speaking, you followed someone for some reason, with that in mind, who that person follows could potentially be someone you also want to follow.

(in the case that it is someone who follows the person you followed, and they didn't follow them back, then it usually is a bad recommendation tbh)

Sometimes there are bad recommendations, but most of the time I find people that I would be interested in if I already knew them.
It makes it easier to discover projects w/o actively looking for them.

If you don't like it so much, then just write a userscript to disable it, it can't be more than 20 lines of JavaScript.

**Reply by ghost** [reply]
`If you don't like it so much, then just write a userscript to disable it, it can't be more than 20 lines of JavaScript.`
Horrible idea, why just not setup own gitlab then. ;)

**Comment by pumba-dev** [comment]
GitHub with a recommendation algorithm.
